models/usersModel.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+function fetchAllUsers() {
+  return db.query(`SELECT * FROM users`).then((users) => {
+    const rows = users.rows;
```

Comment:
might be more efficient for you to destructure rows from your .then argument here, but your way is arguably more semantic